src/main/java/com/CezaryZal/api/body/entity/BodySizeDao.java

```diff
@@ -0,0 +1,21 @@
+package com.CezaryZal.api.body.entity;
+
+import java.time.LocalDate;
+
+public class BodySizeDao extends FormBodySize{
```

Comment:
Poczytaj co to pattern DAO, to co tutaj zrobiłeś to jest bardziej DTO

Reply:
Ok, doczytałem.
Moje DAO nie ma nic wspólnego ze wzorcem.
Usuwam moją encje Dao